nomenclature/processor/region.py

```diff
@@ -165,7 +178,7 @@ def check_native_or_common_regions(cls, values):
         ):
             raise ValueError(
                 "At least one of the two: 'native_regions', 'common_regions' must be"
-                f"given in {values['file']}"
+                f" given in {values['file']}"
```

Comment:
Suggest to change message to "At least one of 'native_regions' and 'common_regions' must be provided in {value['file']}"